Docker/pgbouncer/pgbouncer.ini

```diff
@@ -0,0 +1,12 @@
+[databases]
+adaguc = host=adaguc-db port=5432 dbname=adaguc
+
+[pgbouncer]
+listen_port = 5432
+listen_addr = *
+auth_type = trust
```

Comment:
On the discussion we had, I think this is the interesting bit:
`PgBouncer handles its own client authentication and has its own database of users. These settings control this.`
From: https://www.pgbouncer.org/config.html

We use `auth_type` is trust, which means:
`No authentication is done. The user name must still exist in auth_file.`

So, anyone claiming to be `adaguc` is accepted by pgbouncer. And pgbouncer handles the login with postgres, using the password as provided in `userlist.txt`.
Let me know if this understanding is not correct.